src/VisualStudio/Core/Def/Implementation/AnalyzerDependencyCheckingService.cs

```diff
@@ -20,6 +20,7 @@ namespace Microsoft.VisualStudio.LanguageServices.Implementation
     internal sealed class AnalyzerDependencyCheckingService
     {
         private static readonly object s_dependencyConflictErrorId = new object();
+        private static readonly IAssemblyWhiteList s_systemPrefixWhiteList = new AssemblyNamePrefixWhiteList("System");
```

Comment:
:question: Should this include **mscorlib** as well? See DotNetAnalyzers/StyleCopAnalyzers#1345 for reference.